testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeCloneTableCompatibility.java

```diff
@@ -250,6 +251,75 @@ public void testReadFromSchemaChangedDeepCloneTable()
         testReadSchemaChangedCloneTable("DEEP", false);
     }
 
+    @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    public void testShallowCloneTableMerge()
```

Comment:
It would be good to have similar test for partitioned table and table with deletion vectors